tests/benchdnn/inputs/conv/harness_conv_f32

```diff
@@ -6,3 +6,7 @@
 --dir=BWD_WB --batch=set_conv_all
 --mb=0                      # for bwd_w use the actual mb for 1 topology
 --dir=BWD_WB --batch=shapes_resnet_50
+
+--reset --dt=f32:bf16:f32,f32:f16:f32
```

Comment:
I'd probably move those into `test_conv_bfloat16_nxc` and `test_conv_float16_nxc` instead.